packages/shopify-codemod/transform-cli/create-transform.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable no-sync */
+const fs = require('fs');
+const path = require('path');
+const utils = require('./transform-management-utils');
+
+function createFileSync(filePath, content) {
+  const fd = fs.openSync(filePath, 'wx');
```

Comment:
It's weird because I write directly to file in the rename helper.  Who knows what I was thinking?  Fixed now.